### DISCUSSION TITLE
Equipo 3 - Proyecto Final

### DISCUSSION THREAD

**Discussion by EdittTB** [PatronesHermosos/design-thinking-1ra-generacion#8] [Proyecto Final]
Buenas tardes Equipo 3, lamentablemente no pudimos ponernos de acuerdo con los canales de comunicación para completar las actividades del proyecto, así que abri este hilo para discutir asuntos relacionados. Realicé algunas contribuciones al entregable que inicio @jennyperez28. Me avisan que les parece. Hoy estaré en la sesión interactiva, espero que podamos coincidir.

Atte.

Claudia @EdittTB @

@jennyperez28 
Fabiola

**Comment by jennyperez28** [comment]
Hola compañera, soy Jenny.
Me parece mas facil comunicarnos por otro medio. Si quieres me escribes al whatsApp +573122186253.
Esperemos un próximo encuentro a ver si nuestra otra compañera se pone en contacto o lee esto.

**Reply by Zzariel** [reply]
hola @jennyperez28 no pude agregarte, agregue tu numero pero no me sale la opción del whatsapp.

**Comment by KasturiSShah** [comment]
Hola @jennyperez28 @EdittTB y Fabiola,
Gracias por empezar este hilo. Ví que el formulario en Github está rellenado para la etapa 0. ¿Me avisan por favor si está completada, para poder dar retroalimentación a ustedes?
Gracias,
Kasturi

**Reply by EdittTB** [reply]
Buenas tardes Kasturi, aún no pudimos conectarnos con Fabiola, esperamos el feedback de ella y sus contribuciones. Esperamos que hoy se encuentre en la asesoría. Por el monento no estaría concluido.

**Reply by KasturiSShah** [reply]
Buenas tardes Claudia, 
Me parece un buen plan buscar ella en la asesoría. Si no pudieran charlar ahí, me avisarías por favor, y haré seguimiento con ella.
Kasturi

**Reply by Zzariel** [reply]
@KasturiSShah hola ya me contacte con mis compañeras muchas gracias.

**Reply by KasturiSShah** [reply]
Hola @Zzariel gracias por avisar, me alegre! Kasturi

**Comment by EdittTB** [comment]
Buenos días @KasturiSShah, por favor si el equipo de PH puede revisar nuestro protocolo y dar algunas recomendaciones o sugerencias, se lo agradeceriamos mucho! Gracias!

**Reply by KasturiSShah** [reply]
Hola @EdittTB @jennyperez28 @Zzariel 

Gracias por entregar la etapa 0 del proyecto. Perdón por la tardanza, no me di cuenta que está todo listo, entonces gracias a @EdittTB y @jennyperez28 por el mensaje avisándome al respecto.

He revisado la etapa 0, y me parece un tema muy importante hoy en día y un tema que afecta a muchas personas en todo el mundo. Déjame decir aquí que es un asunto delicado, entonces sería muy importante tratarlo con mucha sensibilidad y empatía. Gracias por elegir este tema para su proyecto, estaría muy interesada en las ideas que generarán al respecto. Mirando a tu protocolo, veo que ustedes han construido preguntas con empatía, ¡buen trabajo!

Tengo algunas sugerencias: sería importante empezar con algo para romper el hielo (una pregunta, broma, historia, lo que sea) antes de recolectar información sobre la edad, genero, nacionalidad, etc de la persona. También avisar al entrevistado/a sobre la duración de la entrevista _antes_ de empezarla. La pregunta 4 _¿Cómo se siente ahora?_ es un poquito general, mi sugerencia sería hacerla más especifica según la respuesta del entrevistado/a para la pregunta 3.  También, la pregunta 4 podría ser difícil contestar, especialmente si alguien está encontrando obstáculos en su salud mental. Entonces, una posibilidad sería tomar tiempo con esta pregunta quizás tener una conversación ahí, antes de pasar a la pregunta 5. Me gusta que las preguntas 5 y 6 se hablan del futuro, y aspectos positivos. Me parece una buena manera de terminar, especialmente durante una entrevista así sobre un tema difícil. Y también, terminar con gratitud para el tiempo que el entrevistado ha invertido en la conversación.

¿Han agendado las entrevistas que cada una de ustedes completarán? Mi sugerencia sería completarlas tan pronto como sea posible, para que tengan tiempo suficiente para organizar sus apuntes.

¿Tienen preguntas para mí? Me avisan cualquier cosa.

¡Mucha suerte!
Saludos,
Kasturi

**Reply by EdittTB** [reply]
Muchas gracias por las orientaciones Kasturi, ahora que ya tenemos el protocolo procedemos con las entrevistas, intentaremos agendar lo más rápido posible! Saludos!!

**Comment by EdittTB** [comment]
Buenas tardes @KasturiSShah, por favor solicitar un nuevo plazo para presentar el entregable de la etapa 1. No todas hemos concluido con las entrevistas, pero si hay un avance por parte de nuestra colega @Zzariel. Hemos actualizado nuestra pasta con el link de nuestro jamboard, pero aún no hemos terminado las entrevistas ni definido el problema. Agradezco tu compresión. 

Saludos! 

@EdittTB 
@jennyperez28 
@Zzariel

**Reply by KasturiSShah** [reply]
Hola @EdittTB @jennyperez28 @Zzariel
Claro que sí, entiendo que agendar entrevistas puede tomar más tiempo que esperábamos. Como hablamos ayer en la asesoría, tienen prórroga hasta este domingo. Cualquier consulta más, aquí ando.
Saludos,
Kasturi

**Comment by EdittTB** [comment]
Buenas noches @KasturiSShah, te escribimos para informarte que el entregable 1 ya está listo para revisión. Sobre el entregable 2, te agradeceríamos si puedes revisar nuestros avances, ya solo nos faltaría el video y algunos ajustes. Si no hay nada que corregir en la etapa 2, podríamos entregar el video hasta el martes? ya que ese día nos reuniremos todas. Muchas gracias por tu comprensión. 

Atte.:

@Zzariel 
@jennyperez28 
@EdittTB

**Reply by KasturiSShah** [reply]
Hola @Zzariel @jennyperez28 @EdittTB

Gracias por entregar la etapa 1. La revisaré el lunes.

Claro, tienen prórroga hasta el martes para la etapa 2.

Feliz fin!
Kasturi

**Reply by EdittTB** [reply]
Muchísimas gracias, buen domingo!!!

**Comment by EdittTB** [comment]
Buenas noches @KasturiSShah, ya terminamos la etapa 2, te agradecemos la retroalimentación. Gracias!!!

Atte.

@jennyperez28 
@Zzariel 
@EdittTB

**Reply by KasturiSShah** [reply]
Hola @jennyperez28 @Zzariel @EdittTB
¡Que bueno que terminaron el trabajo durante la asesoría! Vale, estaré revisando la etapa 2 tan pronto como sea posible.
Kasturi

**Comment by EdittTB** [comment]
Buenas tardes @KasturiSShah, disculpa por escribir esta semana de descanso, solo me gustaría pedirte que nos des un nuevo plazo para mandar el entregable de la etapa 3. No pudimos concluir aún con nuestro prototipo, te agradezco de antemano tu comprensión.

Atte

@jennyperez28 
@Zzariel 
@EdittTB

**Reply by KasturiSShah** [reply]
Hola @jennyperez28 @Zzariel @EdittTB

Claro que sí, ¿hasta que fecha quieren la prórroga? 

Felices vacaciones. Estaré en contacto con retroalimentación sobre la etapa 2 pronto.

Kasturi

**Comment by KasturiSShah** [comment]
Hola @jennyperez28 @Zzariel @EdittTB,

Gracias por entregar la etapa 1 y 2. He revisado las entregas, Jamboard, capturas de pantalla, y el vídeo. Escribo con retroalimentación sobre las etapas 1 y 2. 

Me parece muy interesante juntar sus apuntes sintetizados y las ideas en una página web. Creo que sus ideas son creativas y quizás haya forma de incluir las ideas más creativas en el sitio web. Por ejemplo, musicterapía, montaje de fotografías, y meditación guida. Cada una puede ser una sub-página o una sección en su página web. Incluir estas ideas podría ofrecer maneras diversas y divertidas para involucrarse en el entrenamiento/servicio que ofrecen ustedes.

Una pregunta, al momento de generar las ideas, ¿hicieron la ideación juntas o cada una individualmente? Y ¿luego usaron SCAMPER para organizar las ideas?

¿Cual será el prototipado low fidelity que diseñarán para su etapa 3? Hay varias opciones, por ejemplo, diagrama mostrando la estructura de la página web, o diagrama mostrando su estrategia para juntar/alinear las ideas que destacaron usando SCAMPER. Si quisieran discutir el prototipo conmigo, no duden en escribirme por este medio.

Quedo atenta a su respuesta sobre la nueva fecha límite que quieren para la entrega 3.

No hay prisa para contestar a este mensaje. Que disfruten de la semana.

¡Feliz Semana Santa!
Kasturi

**Reply by EdittTB** [reply]
Buenas noches Kasturi! gracias por el feedback! En efecto, juntamos las ideas generadas para tratar la salud mental de las personas, pero en una aplicación. Sobre las preguntas, si cada una fue generando las ideas desde nuestra experiencia y las experiencias de las personas entrevistadas, como muchas eran iguales fuimos reduciendo la lista. Usamos SCAMPER para "Combinar" las ideas que eran similares en propósito (o agrupar), "Adaptar" ideas para que estén relacionadas a tratar la salud mental y "Eliminar" ideas repetidas. Hoy pudimos reunirnos y definir nuestro prototipo low-fidelity. La entrega de la etapa 3 esta en nuestra carpeta (Slides con los diseños y el video). Te agradecemos mucho tu feedback, así podemos refinar la idea en la siguiente etapa. Muchísimas gracias! Buena semana y buen descanso en el feriado!
Atte
@jennyperez28 
@Zzariel 
@EdittTB

**Reply by KasturiSShah** [reply]
Hola @jennyperez28 @Zzariel @EdittTB

Gracias por explicar su proceso para reducir y organizar las ideas para la etapa 2. Me parece un buen proceso, y siempre es interesante aprender sobre cual(es) letra(s) en "SCAMPER" fueron más útiles para cada equipo.

Revisaré la etapa 3 (diapositivas + vídeo) tan pronto como sea posible.

Saludos,
Kasturi

**Comment by KasturiSShah** [comment]
Hola @jennyperez28 @Zzariel @EdittTB,

Espero estén muy bien y que disfrutaron de los feriados la semana pasada.

Gracias por entregar la etapa 3 de su proyecto final. En primer lugar, felicidades por crear diapositivas tan claras y concisas. Fue un resumen muy interesante de su prototipado low fidelity hasta este momento. ¡Me parece genial y muy útil el app que están creando ustedes!

Tengo dos sugerencias:
1. Para alguien que quiere pedir apoyo presencial para su estado en especifico (por ejemplo, alguien que se siente ansioso), ¿qué le recomendaría el app? Una posibilidad sería compartir una lista de recursos para salud mental en el país donde está, pero lo importante sería identificar a alguien que la va a actualizar frecuentemente. O, identificar a algunos médicos o psicólogos que quieren comprometer a charlar con usuarios del app?
2. Desde mi perspectiva, el éxito del app depende de un poco en crear una comunidad de soporte sin juzgamiento ¿Opinan que es importante moderar la conversación en el app Nutri-Mente de cierta manera? Por ejemplo, cierto tipo de comentarios recibidos. 

Una pregunta de curiosidad: ¿pensaron en incorporar un componente presencial para la comunidad Nutri-Mente, o están pensando en crear una comunidad virtual?

¿Tienen preguntas para mi?

Mucha suerte con la etapa 4 -- ¡adelante!

Saludos a la distancia,
Kasturi

**Comment by KasturiSShah** [comment]
Hola @jennyperez28 @Zzariel @EdittTB,

¿Cómo andan con su proyecto final? ¿necesitan una prórroga para la etapa 4 del proyecto final?

¡La clausura del curso es la semana que viene! Estamos en las últimas estrechas del curso. Sería un placer mirar su presentación de este proyecto con el resto de las alumnxs ahí. Déjame saber si les conviene la asignación de su equipo a una de las dos sesiones de clausura del curso, o si quieren cambiar su día asignado al otro día.

Mi recomendación sería poner 1-2 días de trabajo en completar la etapa 4 y 5, y luego preparar para la presentación final. La etapa 5 no necesita que la etapa 4 sea completada. Dado que hay 3 personas en este equipo, pueden dividir las tareas, por ejemplo: 2 personas completando la etapa 4 (una de ellas que hará la conversación con un usuario y la otra que hará los cambios al prototipado) y una persona completando la etapa 5 (es corta, solo cuestión de rellenar una tabla).

Estamos aquí para apoyar, déjenme saber cualquier cosa.

Mucha suerte, mucho éxito,
Kasturi

**Reply by EdittTB** [reply]
Buenas noches Kasturi, lamentamos la demora en la respuesta, comentarte lo siguiente:
1. Las tres confirmamos nuestra participación el día miércoles para la presentación final.
2. La etapa 4 y la etapa 5 ya están completas.
Cualquier sugerencia o feedback es siempre bienvenido, estamos pendientes.
Muchas gracias por la paciencia.

Atte.:
@Zzariel 
@jennyperez28 
@EdittTB

**Reply by KasturiSShah** [reply]
Hola  @jennyperez28 @Zzariel @EdittTB,

Gracias por el aviso. No hay nada que disculpar.

1. Muy bien, nos vemos en unos días 😄 😄 
2. Gracias por subir las etapas 4 y 5. Las revisaré hoy mismo por la noche. Dado las alturas del curso, recomiendo que pasen tiempo preparando para la presentación durante la clausura. No les daré sugerencias súper grandes sobre las 4 y 5, y no es nada necesario implementar las sugerencias antes de la clausura.

Saludos
Kasturi

**Comment by KasturiSShah** [comment]
Hola @jennyperez28 @Zzariel @EdittTB,

Gracias por entregar las etapas 4 y 5. Las he revisado, me parecen súper interesantes. ¡Están avanzando bien! 

Algunos comentarios:
1. A mí me parece que los comentarios que recibieron desde usuarios fueron muy útiles. Algunas sugerencias (por si quisieran continuar con el proyecto aun cuando el curso se termine):
- _"Lo relevante fue que muchos usuarios consideraron que la idea es buena pero no todos usarían la aplicación, lo que sugiere mayores esfuerzos en el diseño de testes para verificar el concepto y la usabilidad"_ Acá sugiero que el primer paso sea entender **porque** no usarían el app, y luego diseñar tests que prueban las razones. (En vez de diseñar pruebas para el concepto y usabilidad sin saber el razonamiento de los usuarios). Por ejemplo, si indican que usar un app toma mucho tiempo, luego pueden diseñar pruebas que testean cuánto tiempo necesitan los usarios para completar tareas comunes en la interfaz.
- _"Una idea interesante que surgió fue el interés de tener un registro o bitácora de las actividades realizadas en la aplicación en periodos de tiempo, esto tendría un uso como un diario o bitácora con un análisis cuantitativo que permita realizar una evaluación general del estado de animo de la persona a lo largo del tiempo y alertar si existe alguna anomalía."_ Muy interesante la idea, creo que este tipo de estadisticas (agregadas y anónimas) puede ser interesante para organizaciones/doctores trabajando con salud mental también. Puede ser que tener la capacidad de hacer análisis  cuantitativa les apoyará buscar colaboraciones en diferentes áreas.

2. En el prototipo alta fidelidad, el flujo es muy claro. ¡Excelente! Una pregunta: después de picar "Identificar" o "bitácora" en _¿Cómo te sientes hoy?_ ¿qué página/interfaz verá el usuario? No veo esa página en la etapa 4. Solo una sugerencia que piensen en ella (¡no es necesario hacerlo antes de su presentación final el miércoles!)

3. En el spreadsheet que compartieron, me alegre que diseñaron su propia rúbrica. ¡Muy bien! **Sugiero que incluyan la rúbrica en su presentación final, se hace más fuerte el proyecto y la prueba** 😄 
4. En el mismo spreadsheet, veo una sugerencia de integrar el app con ChatGPT ... hay que tener mucho cuidado aquí porque ChatGPT inventa respuestas si no sabe la respuesta (hay muchos artículos sobre este asunto, es interesante) y dado que el proyecto es sobre algo sensible, el uso de ChatGPT puede ser peligroso 
5. Sobre la plantilla Sprint en la etapa 5, es excelente que están pensando en los pasos que se necesitan para crear el app. Algunas sugerencias:
- creo que crear el app tomará más de 2 semanas (aun cuando estén usando una herramienta como Marvel). Hay muchas decisiones que tienen que tomar ... de diseño, estructura, aestetica. Es mejor no apurar y tomar bien el tiempo que necesitan para crear el mejor producto posible
- Sugiero agregar una tarea en la cual identificarán quiénes son los stakeholders a los que presentarán su proyecto
- Sugiero agregar más tiempo para hacer todas las pruebas. Es probable que para cada tipo de prueba (validación y ánalisis de usabilidad / concepto) necesitarán varias rondas para identificar todos los cambios que necesitan los usuarios. También es importante mostrar el producto a varios usuarios, que puede tomar mucho tiempo.
- Sugiero agregar una tarea en la cual identificarán quiénes serán usuarios con que puedan hacer las pruebas. Acá será importante pensar en quiénes van a usar el app (rangos de edades, países,...etc)

Espero les sirvan los comentarios. Déjame re-iterar que no es nada necesario implementar las sugerencias antes de la clausura. Pero recomiendo que incluyan el punto 3 (la rúbrica) en su presentación 😄 

Mucha suerte, me avisan cualquier cosa.
Hasta pronto
Kasturi

**Reply by EdittTB** [reply]
Buenas noches Kasturi!!! solo confirmar que los cambios fueron realizados en las planillas Excel correspondientes y en nuestra presentación. Tambien avisarte que la etapa 6 esta concluida. Por favor considerar que nos excedimos de tiempo en el video, ya que somos 3 personas. Te agradecemos mucho toda la retroalimentación y el apoyo en nuestro proyecto. Hasta una siguiente oportunidad!!!!

Atte.

@jennyperez28 
@Zzariel 
@EdittTB

**Reply by KasturiSShah** [reply]
Hola @jennyperez28 @Zzariel @EdittTB,

Gracias por completar las etapas del proyecto final. No se preocupen si el video es largo, entiendo que hay 3 personas en el equipo. Los 5 minutos son una guía, más para las presentaciones en vivo durante la clausura del curso.

Gracias a ustedes por su dedicación a lo largo del curso. Ha sido un gran placer y un gran gusto compartir este espacio contigo, gracias por todo lo que he aprendido desde ustedes y su proyecto. 

Hasta la siguiente oportunidad, muchos abrazos a la distancia,
Kasturi
